assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java

```diff
@@ -1679,6 +1679,7 @@ public class SimpleSubmission {
         private String gradableId;
         private String submittedText;
         private String dateSubmitted;
+        private Instant dateSubmittedTimestamp;
```

Comment:
I'd call this dateSubmittedEpochMillis, if that is what is produced in the JSON. Then it is self documenting. I would check if it is actually millis or seconds that get serialised into JSON, though. So, it would be dateSubmittedEpochMillis or dateSubmittedEpochSeconds

Reply:
The current instant is serialised into JSON like this:

`"dateSubmittedTimestamp" : {"epochSecond": 1633681328, "nano": 789000000}`

Reply:
Ah, right. In that case the name's perfect. Thanks.

Reply:
What do you think about just using the actual millis in the field and not relying on Jackson's serialisation of the Instant? That might be more predictable and future proof. So, if we moved this to webapi, out of direct, the behaviour would be exactly the same?

Reply:
I think that it would be better as you say, with the last commit the date looks like this: `"dateSubmittedTimestamp": 1659513572`